src/core/display/Container.js

```diff
@@ -628,11 +628,20 @@ Container.prototype.destroy = function (destroyChildren)
 {
     DisplayObject.prototype.destroy.call(this);
 
-    if (destroyChildren)
-    {
-        for (var i = 0, j = this.children.length; i < j; ++i)
-        {
-            this.children[i].destroy(destroyChildren);
+    if (!destroyChildren) {
+    } else {
```

Comment:
Whoops sorry, I think that's a hold over from some older pixi code I was working in. I'll correct it.